### DISCUSSION TITLE
Request to Update OpenSSL-Config Version in Wolfi Base Image

### DISCUSSION THREAD

**Discussion by Naveen-is-here** [wolfi-dev/os#35231] [General]
Hi Team,

I've been using the Wolfi base image and noticed that the current `openssl-config` version (`3.3.0-r5`) in the image is flagged with the following vulnerabilities:

| **Package**         | **Vulnerability**  | **Severity**  | **Fixed In**      |
|----------------------|--------------------|---------------|-------------------|
| openssl-config       | CVE-2024-5535     | Critical      | 3.3.1-r3          |
| openssl-config       | CVE-2024-6119     | High          | 3.3.2-r0          |
| openssl-config       | CVE-2024-4741     | High          | No fix available  |

The latest version (`3.4.0-r2`) appears to address most of these issues and is available in other repositories. Would it be possible to update `openssl-config`  to mitigate these vulnerabilities?

Let me know if I can assist with testing or provide more details.

Thank you for maintaining such an excellent ecosystem!

**Comment by xnox** [comment]
@Naveen-is-here openssl-config package is no longer built nor used nor needed. None of the Changuard images contain it anymore, thus please stop installing it, or if you still have it, remove it. If you require an openssl config for any reason you can touch an empty file; or export `OPENSSL_CONF=/dev/null` environment variable.
